tracing-tracy/src/lib.rs

```diff
@@ -121,14 +124,13 @@ impl<F> TracyLayer<F> {
         error_msg: &'static str,
     ) -> &'d str {
         // From AllocSourceLocation
-        let mut max_len =
-            usize::from(u16::max_value()) - 2 - 4 - 4 - function.len() - 1 - file.len() - 1;
+        let mut max_len = usize::from(u16::MAX) - 2 - 4 - 4 - function.len() - 1 - file.len() - 1;
         if data.len() >= max_len {
             while !data.is_char_boundary(max_len) {
                 max_len -= 1;
             }
             self.client
-                .color_message(error_msg, 0xFF000000, self.stack_depth);
+                .color_message(error_msg, 0xFF00_0000, self.stack_depth);
```

Comment:
Eh, this is a toss-up. Since this is a colour hex, if we’re making it more readable it would make more sense to either `0xFF0000_00` (the alpha "channel" is ignored) or `0xFF_00_00_00` for the 4 channels in RGBx.

Reply:
Looks like this comes from my personal clippy config and not the default, so reverting.